app.js

```diff
@@ -25,22 +30,29 @@ imageConstructor.prototype.addPicture = function(index){
   imageContainer.appendChild(image);
   this.timesShown += 1;
 };
+imageConstructor.prototype.percentage = function(){
```

Comment:
Group the prototype method definitions up with the constructor function itself. They are part of the definition of what instances of your constructor can do.